app/src/hid.c

```diff
@@ -55,7 +55,9 @@ int zmk_hid_unregister_mod(zmk_mod_t modifier) {
             continue;                                                                              \
         }                                                                                          \
         keyboard_report.body.keys[idx] = val;                                                      \
-        break;                                                                                     \
+        if (val) {                                                                                 \
```

Comment:
Arbitrary. I'm happy to switch to inline functions.

Reply:
> Probably not something to change in this PR, but is there a reason these are macros instead of functions?

I was also wondering that for a future refactor PR.